app/createTournament/createTournamentBusiness.js

```diff
@@ -0,0 +1,53 @@
+import * as common from '../../utils/common';
+import * as playerValidation from '../playersinscription/addPlayerBusiness';
+import {Player} from '../Player';
+import * as Math from '../../node_modules/mathjs/dist/math.min';
+
+export function validateNumberofPlayers(){
+
+var numberofPlayers = $('#tourneyPlayersList  >tbody >tr').length;
+  // 2^n in the future
+var validationResult={
+  success:true,
+  error:null
+};
+  if(numberofPlayers<8){
+    validationResult.success=false;
+    validationResult.error = {
+
+      lessthanrequired:true
+    };
+        addGhostPlayer(8-numberofPlayers);
+  }
+  if(numberofPlayers>8){
+    validationResult.success=false;
+    validationResult.error = {
+
+      morethanrequired:true
+    };
+  }
+  return validationResult;
+}
+
+function addGhostPlayer(number){
```

Comment:
Put on other words, this module should be split into a) validation and b) create tournament? I only see ghost players management, no tournament creation.